source/class/cv/Application.js

```diff
@@ -822,6 +853,69 @@ qx.Class.define('cv.Application',
       }
     },
 
+    _checkBackend: function () {
+      const url = cv.io.rest.Client.getBaseUrl().split('/').slice(0, -1).join('/') + '/environment.php';
+      const xhr = new qx.io.request.Xhr(url);
+      xhr.set({
+        method: 'GET',
+        accept: 'application/json'
+      });
+      xhr.addListenerOnce('success', function (e) {
+        const req = e.getTarget();
+        const env = req.getResponse();
+        const [major, minor] = env.phpversion.split('.').map(ver => parseInt(ver));
+        const parts = env.required_php_version.split(' ');
+        const disable = parts.some(constraint => {
+          const match = /^(>=|<|>|<=|\^)(\d+)\.(\d+)$/.exec(constraint);
+          if (match) {
+            const operator = match[1];
+            const majorConstraint = parseInt(match[2]);
+            const minorConstraint = parseInt(match[3]);
+            switch (operator) {
+              case '>=':
+                if (major < majorConstraint || (major === majorConstraint && minor < minorConstraint)) {
+                  return true;
+                }
+                break;
+              case '>':
+                if (major <= majorConstraint || (major === majorConstraint && minor <= minorConstraint)) {
+                  return true;
+                }
+                break;
+              case '<=':
+                if (major > majorConstraint || (major === majorConstraint && minor > minorConstraint)) {
+                  return true;
+                }
+                break;
+              case '<':
+                if (major >= majorConstraint || (major === majorConstraint && minor >= minorConstraint)) {
+                  return true;
+                }
+                break;
```

Comment:
(Without explicit testing:) I guess those rules are not correctly implemented. At least `>=` and `>` as well as `<=` and `<` seem to be wrongly exchanged with each other.

As the environment.php is also exporting `PHP_VERSION_ID`: it might be a simpler way to compare the versions.

Reply:
The comparisons work, please keep in mind that `return true` means the version constraint fails. e.g. `>=7.0` fails when major < 7 OR (major === 7 AND minor < 0)

Using the version id might be a better option, I will check tha.